diff.py

```diff
@@ -132,6 +132,13 @@ def complete_symbol(
         help="""Tweak --source handling to make it work with binutils < 2.33.
         Implies --source.""",
     )
+    parser.add_argument(
+        "-j",
+        "--section",
+        dest="diff_section",
+        metavar="SECTION",
+        help="Diff restricted to a given output section.",
```

Comment:
there's a lot of `diff_section or ".text"`, can we do `default=".text"` here instead?